addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
+        var remainingViewElement = Ember.$('[role="dialog"]:last-child').closest('.ember-view');
```

Comment:
Yes, it will because it's a jquery object and jquery objects are wrappers over an array of elements. Even if no element was found, it's still a wrapper over an empty array so the call will just pass through.